src/index.js

```diff
@@ -26,22 +27,26 @@ class Funnies {
       if (this.record[smallest] > this.record[message]) {
         return message;
       } else if (this.record[smallest] === this.record[message]) {
-        return _.sample([smallest, message]);
+        return sample([smallest, message]);
       } else {
         return smallest;
       }
     });
 
     // update the recrd to show that this message was picked
     this.record[smallestMessage] += 1;
+
     return smallestMessage;
   }
 
   messageHTML() {
     let message = this.message();
-    let html = `<div class="funnies">
-      <span class="loading-funny">${message}</span>
-    </div>`.replace(/(\r?\n|^ +)/gm, '');
+    let html = `
+      <div class="funnies">
+        <span class="loading-funny">${message}</span>
+      </div>
+    `.replace(/(\r?\n|^ +)/gm, '');
```

Comment:
May be out of scope for this PR -- did this to try and read the code better. Willing to remove if an issue.